pennylane_qiskit/qiskit_device.py

```diff
@@ -409,6 +409,7 @@ def batch_execute(self, circuits):
                 self._samples = self.generate_samples(circuit_obj)
 
             res = self.statistics(circuit.observables)
+            res = np.asarray(res)
```

Comment:
```suggestion
            res = np.asarray(self.statistics(circuit.observables))
```

Reply:
Personally, I prefer breaking up statements into multiple lines, such that the smaller steps can be followed easily.